packaging/material/packaging_python.md

```diff
@@ -0,0 +1,180 @@
+---
+type: slide
+slideOptions:
+  transition: slide
+  width: 1400
+  height: 900
+  margin: 0.1
+---
+
+<style>
+  .reveal strong {
+  font-weight: bold;
+    color: orange;
+  }
+  .reveal p {
+    text-align: left;
+  }
+  .reveal section h1 {
+    color: orange;
+  }
+  .reveal section h2 {
+    color: orange;
+  }
+</style>
+
+# How to prepare a Python code for packaging
+
+---
+
+## What is PyPI?
+
+- [PyPI](https://pypi.org/) = **The Python Package Index** is a repository of software developed in the Python community
+- PyPI itself is developed on GitHub through another software called [Warehouse](https://github.com/pypa/warehouse)
+- PyPI has a informative [public dashboard](https://p.datadoghq.com/sb/7dc8b3250-85dcf667bd?from_ts=1636972832484&to_ts=1636976432484&live=true) to show its activity
+- Who maintains PyPI and pip? A big community! For example check out [Donald Stufft](https://twitter.com/dstufft/)
+- A major advantage is the active maintenance of PyPI, which is fueled by a big community. PyPI is also on [Twitter](https://twitter.com/PyPI)
```

Comment:
- You don't have to repeat yourself with regards to the big community. You just stated that in the previous point.
- I don't understand why PyPI's Twitter account is mentioned. I also see projects with Twitter accounts that are not actively maintained.